<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1156</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#043c4c;
border-color: rgb(21, 45, 73);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QListWidget" name="ClassView">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(44, 94, 107);
color : rgb(255, 255, 255);
</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="AddClass">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(159, 216, 107);
color: rgb(21, 45, 73);

</string>
          </property>
          <property name="text">
           <string>+ class</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="FunctionView">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(44, 94, 107);
color : rgb(255, 255, 255);
</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="AddFunction">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #53ba83;
color: rgb(21, 45, 73);
</string>
          </property>
          <property name="text">
           <string>+ function</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="VariablesView">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(44, 94, 107);
color : rgb(255, 255, 255);
</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(5, 155, 154);
color: rgb(21, 45, 73);</string>
          </property>
          <property name="text">
           <string>  Variables   </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="CustomGraphicsView" name="StagingArea"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,2,0,0,0">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="SearchBar" name="searchBar"/>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="styleSheet">
         <string notr="true">color: rgb(189, 221, 235)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color : rgb(189, 221, 235)</string>
        </property>
        <property name="text">
         <string>Variables</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listVars">
        <property name="styleSheet">
         <string notr="true">color : rgb(189, 221, 235)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1156</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(21, 45, 73);
color: rgb(220, 234, 251);</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionRestart"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>&amp;Build</string>
    </property>
    <addaction name="actionCompile"/>
    <addaction name="actionRun"/>
    <addaction name="actionOpen_Code"/>
    <addaction name="actionFormat_Code"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBuild"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCompile">
   <property name="text">
    <string>&amp;Compile</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>&amp;Run</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionOpen_Code">
   <property name="text">
    <string>Open Code</string>
   </property>
  </action>
  <action name="actionFormat_Code">
   <property name="text">
    <string>Format Code</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>./headers/CustomGraphicsView.h</header>
  </customwidget>
  <customwidget>
   <class>SearchBar</class>
   <extends>QLineEdit</extends>
   <header>./headers/SearchBar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
